everyBot/cogs/math/math.py

```diff
@@ -0,0 +1,237 @@
+import discord
+from discord.ext import commands
+from discord.ext.commands import has_permissions
+from pymongo.errors import ServerSelectionTimeoutError
+
+from everyBot.cogs import database
+import requests
+
+
+""" Disabled Check """
+async def check_disabled(ctx):
+    try:
+        disabled_commands = await database.fetch_guild_disabled_commands(ctx.guild.id)
+    except ServerSelectionTimeoutError as e:
+        embed = discord.Embed(
+            title="Failed checking command",
+            colour=discord.Color.red(),
+            description=f"Could not check if command is disabled: { e }"
+        )
+        return await ctx.send(embed=embed)
+
+    return ctx.command.name not in disabled_commands
+
+class Math(commands.Cog, name='Math'):

```

Comment:
could the name please be lowercase? - I need to go through and do that for all of the commands

Reply:
Done.

Reply:
Done again...